share/sql/share.go

```diff
@@ -215,7 +216,12 @@ func (m *ShareMgr) UpdateShare(ctx context.Context, ref *collaboration.ShareRefe
 }
 
 func (m *ShareMgr) ListShares(ctx context.Context, filters []*collaboration.Filter) ([]*collaboration.Share, error) {
-	shares, err := m.ListModelShares(ctx, filters)
+	user, ok := appctx.ContextGetUser(ctx)
+	if !ok {
```

Comment:
if `!ok` then the user will by default be nil, so this check is not necessary ;)